docs/src/pages/power-profiler.js

```diff
@@ -0,0 +1,387 @@
+import React, { useState } from "react";
+import classnames from "classnames";
+import Layout from "@theme/Layout";
+import styles from "./styles.module.css";
+import { useInput } from "../utils/hooks";
+import "../css/power-profiler.css";
+import { zmkBoards } from "../data/power";
+import PowerEstimate from "../components/power-estimate";
+
+function PowerProfiler() {
+  const { value: board, bind: bindBoard } = useInput("");
+  const { value: split, bind: bindSplit } = useInput(false);
+  const { value: batterySize, bind: bindBatterySize } = useInput(110);
+
+  const { value: psuType, bind: bindPsuType } = useInput("");
+  const { value: outputV, bind: bindOutputV } = useInput(3.3);
+  const { value: quiescent, bind: bindQuiescent } = useInput(55);
+  const { value: otherQuiescent, bind: bindOtherQuiescent } = useInput(0);
+  const { value: efficiency, bind: bindEfficiency } = useInput(0.9);
+
+  const { value: bondedQty, bind: bindBondedQty } = useInput(1);
+  const { value: percentAsleep, bind: bindPercentAsleep } = useInput(0.5);
+
+  const { value: glowEnabled, bind: bindGlowEnabled } = useInput(false);
+  const { value: glowQuantity, bind: bindGlowQuantity } = useInput(10);
+  const { value: glowBrightness, bind: bindGlowBrightness } = useInput(1);
+
+  const { value: displayEnabled, bind: bindDisplayEnabled } = useInput(false);
+  const { value: displayType, bind: bindDisplayType } = useInput("");
+
+  const [disclaimerAcknowledged, setDisclaimerAcknowledged] = useState(
+    typeof window !== "undefined"
+      ? localStorage.getItem("zmkPowerProfilerDisclaimer") === "true"
+      : false
+  );
+
+  const currentBoard =
+    board === "custom"
+      ? {
+          powerSupply: {
+            type: psuType,
+            outputVoltage: outputV,
+            quiescent,
+            efficiency,
+          },
+          otherQuiescent,
+        }
+      : zmkBoards[board];
+
+  return (
+    <Layout
+      title={`ZMK Power Profiler`}
+      description="Estimate your keyboard's power usage and battery life on ZMK."
+    >
+      <header className={classnames("hero hero--primary", styles.heroBanner)}>
+        <div className="container">
+          <h1 className="hero__title">ZMK Power Profiler</h1>
+          <p className="hero__subtitle">
```

Comment:
what does `hero__` refer tohere?

Reply:
Docusaurus's class name. Reused from the home page.